File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        long total = 0;
        int temp = 0;
        if (a > b) {
            temp = a;
            a = b;
            b = temp;
        }
        for (int i = a; i < b+1; i++) {
            total += i;
        }
        
        return total;
    }
}

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.js
function solution(a, b) {
    const start = Math.min(a,b);
    const end = Math.max(a,b);
    
    let answer = 0;
    
    for (let i = start; i < end + 1; i++) {
        answer += i;
    }
    
    return answer;
}